rskj-core/src/main/java/co/rsk/rpc/netty/Web3ResultHttpResponseHandler.java

```diff
@@ -57,6 +58,7 @@ protected void channelRead0(ChannelHandlerContext ctx, Web3Result msg) {
 
     @Override
     public void channelReadComplete(ChannelHandlerContext ctx) {
+        MDC.clear();
```

Comment:
not needed?

Reply:
I used this method to clear the rpcReqId tag from the MDC. I should MDC.remove(.. instead but maybe this is not the best place.